README.md

```diff
@@ -279,6 +279,37 @@ ORDER BY t ASC
 
 ---
 
+## Templating
+ 
+### Query Variable
+ 
+If you add a template variable of the type `Query`, you can write a ClickHouse query that can
+return things like measurement names, key names or key values that are shown as a dropdown select box.
+ 
+For example, you can have a variable that contains all values for the `hostname` column in a table if you specify a query like this in the templating variable *Query* setting.
+ 
+```sql
+SELECT hostname FROM host
+```
+ 
+To use time range dependent macros like `$timeFilterColumn($column)` in your query the refresh mode of the template variable needs to be set to *On Time Range Change*.
```

Comment:
What is the `timeFilterColumn` macros? I can't find it's definition

Reply:
It's from this PR https://github.com/Vertamedia/clickhouse-grafana/pull/94 Originally it was `timeFilterColumn` and we changed it to `timeFilterByColumn`. Fixing. Good catch